README.md

```diff
@@ -102,20 +101,6 @@ It is also possible to load the bundled css files from a CDN:
 If a specific version is needed, insert the version number into the URL, ie.:
 `<link rel="stylesheet" type="text/css" href="https://unpkg.com/@youseedk/dna@1.0.0/ys-bundle.min.css" />` will load version 1.0.0.
 
-## Browser Support

```

Comment:
I don't think we should delete this section completely, but maybe we could just add a link that points to the browser-support page on dna.yousee.dk?

Reply:
Yeah. I agree